README.md

```diff
@@ -91,6 +92,10 @@
 
 ## Programming Languages
 
+### Arduino
+
+* [nixduino](https://github.com/boredom101/nixduino) - Nix based tool to help build arduino sketches.
```

Comment:
Two small changes to the description:

- Add a hyphen/dash between "Nix" and "based".
- Arduino [should be capitalized](https://www.arduino.cc/en/Main/Policy) when used (had to look that up).